ethereum/executionlayer/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
   testImplementation testFixtures(project(':ethereum:spec'))
   testImplementation testFixtures(project(':infrastructure:async'))
   testImplementation testFixtures(project(':infrastructure:time'))
+  testImplementation project(':services:executionengine')
```

Comment:
This is adding a circular dependency between `services:executionengine` and `ethereum:executionlayer`. We get away with it in IntelliJ and gradle because it's different source trees within the modules but it will cause problems for eclipse and it's not a great practice.

Not sure exactly what the requirement for this is but we need to work out how to avoid this dependency.